cli/tools/task.rs

```diff
@@ -294,6 +302,97 @@ impl ShellCommand for NpmPackageBinCommand {
   }
 }
 
+/// Runs a module in the node_modules folder.
+#[derive(Clone)]
+struct NodeModulesFileRunCommand {
+  command_name: String,
+  path: PathBuf,
+}
+
+impl ShellCommand for NodeModulesFileRunCommand {
+  fn execute(
+    &self,
+    context: ShellCommandContext,
+  ) -> LocalBoxFuture<'static, ExecuteResult> {
+    let mut args = vec![
+      "run".to_string(),
+      "--ext=js".to_string(),
+      format!("--internal-bin-name={}", self.command_name),
+      "-A".to_string(),
+      self.path.to_string_lossy().to_string(),
+    ];
+    args.extend(context.args);
+    let executable_command =
+      deno_task_shell::ExecutableCommand::new("deno".to_string());
```

Comment:
This will use the system `deno` binary and not the current binary?

Reply:
deno_task_shell has some code that will always make the "deno" command use the current executing binary (for the time being, the tests would fail if that weren't the case)